### DISCUSSION TITLE
getting twitter api keys

### DISCUSSION THREAD

**Discussion by altynbek132** [robertodoering/harpy#430] [Q&A]
Hello, I requested Twitter API keys. Twitter support asked me to provide details of my use case (see full email below). What should I reply?

> Hello,
> 
> Thanks for your interest in building on Twitter. 
> 
> Before we can finish our review of your developer account application, we need some more details about your use case. 
> 
> The types of information that are valuable for our review include:
> The core use case, intent, or business purpose for your use of the Twitter APIs.
> If you intend to analyze Tweets, Twitter users, or their content, share details about the analyses you plan to conduct, and the methods or techniques. 
> If your use involves Tweeting, Retweeting, or liking content, share how you’ll interact with Twitter accounts, or their content.
> If you’ll display Twitter content off of Twitter, explain how, and where, Tweets and Twitter content will be displayed with your product or service, including whether Tweets and Twitter content will be displayed at row level, or aggregated.
> Just reply to this email with these details. Once we’ve received your response, we’ll continue our review. We appreciate your help! 
> 
> Thanks, 
> 
> Twitter

**Comment by robertodoering** [comment]
Just answer the questions truthfully, it doesn't really matter
